README.md

```diff
@@ -1 +1,140 @@
-# Catalog-of-my-things
\ No newline at end of file
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#triangular_flag_on_post-deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [❓ FAQ](#faq)
+- [📝 License](#license)
+
+# 📖 [Catalog-of-my-things] <a name="about-project"></a>
+
+**[Catalog-of-my-things]**  Is a console app that will help you to keep a record of different types of things you own: books, music albums, movies, and games. Everything was based on the UML class diagram presented. The use data will be preserved in a json file.
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+<details>
+  <summary>Database</summary>
+  <ul>
+    <li><a href="https://www.ruby-lang.org/en/">RUBY</a></li>
+  </ul>
+
+</details>
+
+### Key Features <a name="key-features"></a>
+
+- **[keep a record of different types of things you own: books, music albums, movies, and games.]**
+- **[List all your data when the option is selected]**
+- **[List all your data when the option is selected]**
+
+### Live Demo <a name="live-demo"><a/>
+
+[Live Demo Link]() is currently not available :exclamation: 

```

Comment:
- This should be marked as an [OPTIONAL] since the details of the readme are actually upon the students and not a must requirement. Also the Readme clearly states that the project live is actually not available ❗.

<img width="439" alt="Screenshot 2023-03-02 at 6 01 43 PM" src="https://user-images.githubusercontent.com/44798044/222435959-e12812c9-a33f-4c79-a564-c54fde9dfae1.png">